### ISSUE TITLE
Consider making multi material dialog optional on 3mf export

### ISSUE THREAD

**Issue by pks5** [openscad/openscad-playground#91]
I like the multi material dialog feature. But in my case, colors are used to highlight certain model features, and are usually not supposed to be printed. 

It would be nice if there would be a setting whether to show the dialog on export 3mf. It still can be accessed via the dropdown if required.

**Comment by ochafik** [comment]
Once you open the dialog and clear the list of materials / leave it empty + export / save the setting, it shouldn’t prompt for it when exporting to 3mf. This should all be persisted in the state / url fragment.

**Comment by pks5** [comment]
True, but if you have multiple playgrounds in iframes like i have, the user is getting prompted each time again.

**Comment by ochafik** [comment]
@pks5 How are you building your iframe URLs?

The list of materials (undefined at first, then empty array if the dialog is closed/saved without adding a color) is encoded in the hash fragment, so it's effectively already a setting.

**Comment by pks5** [comment]
Oh OK, nice to know. I will try that.
